modules/ccs-aws-provisioned.adoc

```diff
@@ -84,3 +84,11 @@ image::VPC-Diagram.png[VPC Reference Architecture]
 == Security groups
 
 AWS security groups provide security at the protocol and port-access level; they are associated with EC2 instances and Elastic Load Balancing. Each security group contains a set of rules that filter traffic coming in and out of an EC2 instance. You must ensure the ports required for the link:https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-other-infrastructure_installing-aws-user-infra[{OCP} installation] are open on your network and configured to allow access between hosts.
+
+[id="osd-security-groups-custom_{context}"]
+=== Additional custom security groups
+When you create a cluster using an existing non-managed VPC, you can add additional custom security groups during cluster creation. Custom security groups are subject to the following limitations:
```

Comment:
```suggestion
When you create a cluster by using a non-managed VPC, you can add custom security groups during cluster creation. Custom security groups are subject to the following limitations:
```
- Added "by" to "using" per IBM Style Word Usage: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u
- You can almost always remove "existing" and the content will retain its meaning
- Similarly, you can simplify "add additional" to "add" and retain the same meaning

Reply:
applied and found same content in this file: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc which I updated